velox/functions/lib/Slice.cpp

```diff
@@ -200,22 +203,40 @@ class SliceFunction : public exec::VectorFunction {
     return std::min(endIndex - start, length);
   }
 };
+} // namespace
+
+/// @tparam IndexKind The `TypeKind` of the function `Slice` start and length
```

Comment:
IndexKind -> Kind

Suggestion:
"The type kind for start and length argument."